### PR TITLE
XW-879 | Introduce Ember CLI

### PR DESCRIPTION
**Pull request by hakubo**
https://wikia-inc.atlassian.net/browse/XW-879

### CODE REVIEW

File: front/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# compiled output
+vendor/
```
Comment (rogatty) [inline front/.gitignore]: There is no `front/vendor`.

File: front/common/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	svgSymbols = require('gulp-svg-symbols'),
+	compile = require('../../gulp/utils/compile-es6-modules'),
+	environment = require('../../gulp/utils/environment'),
+	log = require('../../gulp/utils/logger'),
+	piper = require('../../gulp/utils/piper'),
+	paths = require('../../gulp/paths'),
+	pathsCommon = paths.common,
+	Server = require('karma').Server;
+
+gulp.task('test-common', ['build-common-vendor'], function (done) {
```
Comment (rogatty) [inline front/common/gulpfile.js]: @tomasznapieralski why does it depend on `build-common-vendor` but not on `build-common`?

Comment (tomasznapieralski) [inline front/common/gulpfile.js]: it uses some dependencies like jquery only during tests.
you need to run tests on already builded app

**Comment by hakubo** [comment]
yay!

**Pull request merged**